Dockerfile

```
@@ -13,6 +13,8 @@ FROM base AS frontend-builder
 WORKDIR /app
 RUN pnpm install --frozen-lockfile
 COPY packages/shared/ ./packages/shared/
+# Build shared package first (required for frontend build)
+RUN cd packages/shared && pnpm build
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)